frontend/src/components/App/App.js

```diff
@@ -7,8 +7,8 @@ import audioStartup from '../../audioStartup';
 import playSounds from '../../playSounds';
 import DataVisualiser from '../DataVisualiser/DataVisualiser.js';
 import { Routes, Route, Link } from "react-router-dom";
-const { abridgeData, quantiseData } = require('../../processTubeData');
-const { arrivalEffectTransform, arrivalEffectCreate } = require('../../arrivalEffects');
+import { separateDataIntoLines, fudgeData, abridgeData, quantiseData } from '../../processTubeData';
```

Comment:
I've changed these to import statements just so we are standardised across the site and going off of the format we were given in acebook template etc..